src/test/scala/mesosphere/mesos/TaskBuilderTest.scala

```diff
@@ -365,15 +360,90 @@ class TaskBuilderTest extends MarathonSpec with Matchers {
       } else None
     }
 
-    assert(taskInfo.getContainer.getVolumesList.size > 0, "check that container has volumes declared")
-    assert(taskInfo.getContainer.getDocker.hasVolumeDriver, "docker spec should define a volume driver")
-    assert(taskInfo.getContainer.getDocker.getVolumeDriver == "ert", "docker spec should choose ert driver")
+    assert(taskInfo.getContainer.getVolumesList.size == 2, "check that container has volumes declared")
+    assert(!taskInfo.getContainer.getDocker.hasVolumeDriver, "docker spec should not define a volume driver")
+
```

Comment:
i think rewriting the assertions should be handled for the entire file in a separate PR, not this one